<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmallHoleFillerWidget</class>
 <widget class="QMainWindow" name="SmallHoleFillerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Poisson Editing</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btnFill">
        <property name="text">
         <string>Fill</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkShowInput">
        <property name="text">
         <string>Input</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkShowMask">
        <property name="text">
         <string>Mask</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkShowOutput">
        <property name="text">
         <string>Output</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenImage"/>
    <addaction name="actionOpenMask"/>
    <addaction name="actionSaveResult"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpenImage">
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
  <action name="actionSaveResult">
   <property name="text">
    <string>Save Result</string>
   </property>
  </action>
  <action name="actionOpenMask">
   <property name="text">
    <string>Open Mask</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
